routes/hibp.js

```diff
@@ -6,17 +6,18 @@ const bodyParser = require("body-parser");
 const router = express.Router();
 const jsonParser = bodyParser.json();
 
-const DBUtils = require("../db/utils");
+const DB = require("../db/db");
 
 router.post("/notify", jsonParser, async (req, res) => {
   try {
     const hashes = req.body.hashSuffixes.map(suffix=>req.body.hashPrefix + suffix);
     console.log("hashes: ", hashes);
 
-    const subscribers = await DBUtils.getSubscribersByHashes(hashes);
+    const subscribers = await DB.getSubscribersByHashes(hashes);
     console.log("subscribers: ", subscribers);
 
-    const breach = DBUtils.getBreachByName(req.body.breachName);
+    const reqBreachName = req.body.breachName;
```

Comment:
Do we need a `.toLowerCase()` here as well? (I see a similar pattern below in routes/home.js:10-11)

Reply:
Good catch!